src/lib/ngsiNotify/Notifier.h

```diff
@@ -25,33 +25,37 @@
 *
 * Author: Fermin Galan
 */
-
 #include <map>
 #include <pthread.h>
 
+#include "common/NotificationFormat.h"
 #include "ngsi9/NotifyContextAvailabilityRequest.h"
 #include "ngsi10/NotifyContextRequest.h"
+#include "ngsiNotify/ThreadData.h"
 
-#include "ThreadData.h"
 
-class Notifier {
 
+/* ****************************************************************************
+*
+* Notifier -
+*/
+class Notifier
+{
 public:
-   
-    virtual ~Notifier(void);
-
-    virtual void sendNotifyContextRequest(NotifyContextRequest* ncr,
-                                          const std::string&    url,
-                                          const std::string&    tenant,
-                                          const std::string&    xauthToken,
-                                          const std::string&    fiwareCorrelator,
-                                          Format                format   = JSON);
-
-    virtual void sendNotifyContextAvailabilityRequest(NotifyContextAvailabilityRequest* ncr,
-                                                      const std::string&                url,
-                                                      const std::string&                tenant,
-                                                      const std::string&                fiwareCorrelator,
-                                                      Format                            format           = JSON);
+  virtual ~Notifier(void);
+
+  virtual void sendNotifyContextRequest(NotifyContextRequest* ncr,
+                                        const std::string&    url,
+                                        const std::string&    tenant,
+                                        const std::string&    xauthToken,
+                                        const std::string&    fiwareCorrelator,
+                                        NotificationFormat    notifyFormat = NGSI_V2_NORMALIZED);
```

Comment:
The default value is tricky in this issue.
Clear is that if a v2 request comes in without attrsFormat, the default is 'normalized', and if v1, the default is 'JSON'.
In other places, like extracting a sub from mongo without 'format' it's a little more difficult.

That said, this part is your territory, if you think NGSI_V1_JSON is better here, who am I to argue? :-)
[ Unless the functests start to protest ... ]